theme_kaizen/static/src/scss/base/_fonts.scss

```diff
@@ -0,0 +1,23 @@
+// -----------------------------------------------------------------------------
+// This file contains all @font-face declarations, if any.
+// -----------------------------------------------------------------------------
+@font-face {
+    font-family: "ubuntu";
+    src: url("/theme_kaizen/static/font/Ubuntu-Light.ttf");
+    font-weight: 300;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: "ubuntu";
+    src: url("/theme_kaizen/static/font/Ubuntu-Bold.ttf");
+    font-weight: 700;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: "ubuntu-mono";
+    src: url("/theme_kaizen/static/font/UbuntuMono-Regular.ttf");
+    font-weight: 300;
+    font-style: normal;
+}
```

Comment:
Since there's a `Customize Theme` page for fonts as well, which is now full of `Roboto`, it makes sense to refactor and place `Ubuntu` there, while making it selectable to the user. Perhaps also something for the roadmap.